server/graphql/v2/input/CommentCreate.js

```diff
@@ -1,15 +1,22 @@
 import { GraphQLString, GraphQLInt, GraphQLInputObjectType } from 'graphql';
+import { ExpenseReferenceInput } from './ExpenseReferenceInput';
 
 /**
  * Input type to use as the type for the comment input in createComment mutation.
  */
 const CommentCreate = new GraphQLInputObjectType({
   name: 'CommentCreate',
   fields: () => ({
-    markdown: { type: GraphQLString },
+    markdown: { type: GraphQLString, deprecationReason: '2020-02-26: Please use html' },
     html: { type: GraphQLString },
-    ExpenseId: { type: GraphQLInt },
-    UpdateId: { type: GraphQLInt },
```

Comment:
Are we dropping `UpdateId`?

Reply:
Right now it's not possible to add comments on updates, this was added in the expectation that we should be able to do it in the future. We'll revisit once comments on updates will be possible.